Assets/HoloToolkit/Utilities/Scripts/SortingLayerOverride.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System.Collections;
+
+namespace HoloToolkit.Unity
+{
+    // Overrides the sorting layer of all renderers and child renderers
+    public class SortingLayerOverride : MonoBehaviour
```

Comment:
Should this be a monobehaviour or maybe an extension?

Reply:
Hmm. It could be an extension but I'd probably still want a monobehaviour to call into that extension method as I tend to just add this behavior to gameObject or prefab that I need to sort.